storage.py

```
@@ -0,0 +1,98 @@
+            # Upload with overwrite if exists
+            response = self.supabase.storage. \
+                from_(self.bucket). \
+                upload(object_name, file_data, {'x-upsert': 'true'})
+            
+            if response.get('error'):
+                raise Exception(response['error']['message'])
+            
+            logger.info(f"Uploaded {object_name} to {self.bucket}")
+            return True
+            
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)